aten/src/ATen/core/dispatch/Dispatcher.cpp

```diff
@@ -1,8 +1,87 @@
 #include <ATen/core/dispatch/Dispatcher.h>
 
 namespace c10 {
+
+namespace detail {
+class RegistrationListenerList final {
```

Comment:
docs?

Reply:
This listener business seems more complicated than it ought to be...

Reply:
This is a small and private class in a .cpp file. Don't think it needs docs.

Do you have ideas on how to make the listener stuff easier? My original approach was to have a global static object that iterates over all c10 ops in its constructor and adds them to JIT. Unfortunately, c10 ops are also registered at static initialization time and there is no guarantee for static initialization order, so some c10 ops might be registered after the others were added to JIT. Also, some time into the runtime of the application, somebody might load a new .so library with new schemas and they should also be registered with JIT. That's why I implemented these listeners.